src/Commands/generate/Generators/Migrate/MigrationGenerator.php

```diff
@@ -22,9 +23,9 @@ class MigrationGenerator extends BaseGenerator
      */
     protected function interact(InputInterface $input, OutputInterface $output)
     {
-        $questions = Utils::defaultQuestions() + Utils::defaultPluginQuestions() + [
```

Comment:
We are losing the defaultQuestions() call. Its needed to get machine_name and thus destination, I think.